### DISCUSSION TITLE
Exploring Contributions to Enhance learningequality

### DISCUSSION THREAD

**Discussion by Rathinkiran** [learningequality/kolibri#11521] [General]
Respected Sir/Madam

My name is S.Rathin Kiran a second year undergraduate in Bangalore and I am excited to embark on my journey into the world of open source development. While I am completely new to this open source, I have gained a solid understanding of Python,C and Django, and I am pretty curious to apply my knowledge to contribute meaningfully to open source projects.

I am eager to take on new challenges and make a positive impact within the open source community.so please guide me through this

**Comment by MisRob** [comment]
Hello dear @Rathinkiran, welcome to the community! Thank you for introducing yourself.

We currently have contributing opportunities in three repositories. You can see the contributing guidelines for each of them here:

Kolibri: https://github.com/learningequality/kolibri/blob/release-v0.16.x/CONTRIBUTING.md
Kolibri Design System: https://github.com/learningequality/kolibri-design-system/blob/main/CONTRIBUTING.md
Studio: https://github.com/learningequality/studio (the contributing guidelines are not ready yet here, but you could find issues in the same manner like for the other two repositories)

So, the first step would be to choose an issue to work on in one of those repositories, ask for an assignment, and we can take it from there. It's all described in the guidelines so please have a look. If you have any further questions, do not hesitate to contact us here.

Be well
Michaela

**Reply by Rathinkiran** [reply]
Thank you for this warm gesture and guiding me sir...i will look upon the issue and proceed which is suitable for me

**Reply by MisRob** [reply]
You're welcome!

**Comment by muditchoudhary** [comment]
Hi @Rathinkiran I have a suggestion for you regarding contributing for the first time.

I think you could first start by setting up the Kolibri locally. You can follow these docs: https://kolibri-dev.readthedocs.io/en/develop/getting_started.html

Second I suggest you could read the Kolibri user docs to understand how Kolibri works, the different roles in Kolibri, etc.
https://kolibri.readthedocs.io/en/latest/index.html

Third, you can look for the good first issue or help wanted.

If you stuck somewhere feel free to ask up, we can help.

**Reply by Rathinkiran** [reply]
Hello @muditchoudhary sir 
Thank you for this suggestion and guidance sir.I know the git and github upto a intermediate level and i am familiarize
 with setting up the projects locally but still i will definitely go through the docs once again and then approach

**Reply by MisRob** [reply]
Thank you @muditchoudhary. Just one additional note, you don't need to study all docs, @Rathinkiran. It's a large resource. Just skimming it briefly so you know what type of information is available will be sufficient. Later, you can study in more detail sections that are relevant to your task.

**Reply by Rathinkiran** [reply]
Sure sir @MisRob. Thank you sir, currently i am in that process and taking my own time to go through all the docs and understanding the vision and mission of the organization post which i will step into the field of contributing

Thanks for the guidance sir..
